src/commands/rm.rs

```diff
@@ -43,17 +45,24 @@ pub fn rm(args: CommandArgs) -> Result<OutputStream, ShellError> {
         file => full_path.push(file),
     }
 
-    if full_path.is_dir() {
-        if !args.has("recursive") {
-            return Err(ShellError::labeled_error(
-                "is a directory",
-                "",
-                args.call_info.name_span.unwrap(),
-            ));
+    for entry in glob(&full_path.to_string_lossy()).expect("Failed to read glob pattern") {
```

Comment:
.expect(..) will panic if the value isn't Ok. Instead of panicking, we may just want to return a Err() that tells the user that the glob they sent couldn't be used